README.md

```diff
@@ -46,6 +46,26 @@ Please follow the steps below to build message broker from source code.
 
 Please refer to the document on [message broker architecture](docs/architecture.md) for more information on the internal design.
 
+## High Availability
+
+Support for high availability is available with the message broker.
```

Comment:
Shall we rephrase this line?
Maybe along the lines of `Message broker supports high availability through configuring fail-over nodes`